### PR TITLE
feat: add new zksync farms

### PR DESCRIPTION
**Pull request by ChefBingbong**
### Summary
🌾🚀🌈

This pull request adds support for three new low-fee liquidity pools and a new token on the zkSync layer 2 chain. It updates the `farmsV3` and `zksyncTokens` arrays in `packages/farms/constants/324.ts` and `packages/tokens/src/324.ts` respectively.

> _New pools for `farmsV3`_
> _zkSync scales with low fees_
> _Autumn harvest time_

### Walkthrough
*  Add three new liquidity pools for zkSync-supported pairs ([link](https://github.com/pancakeswap/pancake-frontend/pull/8046/files?diff=unified&w=0#diff-02f301415ffe755f42d9f7b6cf06c434ba122dc41c6e94a77e55ce9e389d62e6R55-R75))
*  Add new token for wrapped ETH on zkSync ([link](https://github.com/pancakeswap/pancake-frontend/pull/8046/files?diff=unified&w=0#diff-a71af7ff44f6a2151f59a9720430da1d5c2a5d5c45623bbeba2d51e4acb77d56R27-R34))

### CODE REVIEW

**Comment by ChefMomota** [comment]
We go 4 farm

) WETH-USDC
pid: 8
fee tier: 0.01%
LP address: 0x7c0e7d6066af191977a4483b445b5c06fc41ecd6

2) BUSD-USDT
pid: 9
fee tier: 0.01%
LP address: 0x38848d93a410446848ca55fdc777fe0b2c30b071

3) rETH-ETH
pid: 10
fee tier: 0.05%
LP address: 0x3693ec2590e6bf8f221f61776dc9274aed7056d6

4) USD+-USDC 
pid: 11
fee tier: 0.01%
LP address: 0x6a8fc7e8186ddc572e149dfaa49cfae1e571108b

File: packages/farms/constants/324.ts [hunk 1]
```diff
@@ -52,4 +52,25 @@ export const farmsV3 = defineFarmV3Configs([
...
+  {
+    pid: 8,
+    lpAddress: '0x662cD659F91528FF27D7Cb6Ac25e6EBA11c4003C',
+    token0: zksyncTokens.weth,
+    token1: zksyncTokens.usdc,
+    feeAmount: FeeAmount.LOWEST,
+  },
+  {
+    pid: 9,
+    lpAddress: '0x38848d93a410446848ca55fdc777fe0b2c30b071',
```
Comment (ChefMomota) [inline packages/farms/constants/324.ts:64]: Unchecksumed address

File: packages/farms/constants/324.ts [hunk 2]
```diff
@@ -52,4 +52,25 @@ export const farmsV3 = defineFarmV3Configs([
...
+    token0: zksyncTokens.busd,
+    token1: zksyncTokens.usdc,
+    feeAmount: FeeAmount.LOWEST,
+  },
+  {
+    pid: 10,
+    lpAddress: '0x3693ec2590e6bf8f221f61776dc9274aed7056d6',
```
Comment (ChefMomota) [inline packages/farms/constants/324.ts:71]: Unchecksumed address

File: packages/tokens/src/324.ts [hunk 3]
```diff
@@ -24,4 +24,12 @@ export const zksyncTokens = {
     'Binance USD',
     'https://www.paxos.com/busd/',
   ),
+  reth: new ERC20Token(
+    ChainId.ZKSYNC,
+    '0x32Fd44bB869620C0EF993754c8a00Be67C464806',
+    18,
+    'rETH',
+    'RETH',
```
Comment (ChefMomota) [inline packages/tokens/src/324.ts:32]: Rocket Pool ETH

File: packages/farms/constants/324.ts (hunk 2 above)
Comment (ChefBingbong) [reply packages/farms/constants/324.ts:71]: sorry linter did not flag my side on vscode

File: packages/farms/constants/324.ts [hunk 4]
```diff
@@ -52,4 +52,25 @@ export const farmsV3 = defineFarmV3Configs([
     token1: zksyncTokens.usdc,
     feeAmount: FeeAmount.LOWEST,
   },
+  {
+    pid: 8,
+    lpAddress: '0x662cD659F91528FF27D7Cb6Ac25e6EBA11c4003C',
```
Comment (ChefMomota) [inline packages/farms/constants/324.ts:57]: Wrong lpAddress 
pid 8, should be `0x7c0e7d6066af191977a4483b445b5c06fc41ecd6`

and make the data above pid1 , should be

[{pid: 11}, pid: 10}, {pid: 9}, {pid: 8}, {pid: 1}, ]

File: packages/farms/constants/324.ts [hunk 5]
```diff
@@ -52,4 +52,25 @@ export const farmsV3 = defineFarmV3Configs([
...
+    token0: zksyncTokens.busd,
+    token1: zksyncTokens.usdc,
+    feeAmount: FeeAmount.LOWEST,
+  },
+  {
+    pid: 10,
+    lpAddress: '0x3693ec2590e6bf8f221f61776dc9274aed7056d6',
+    token0: zksyncTokens.reth,
+    token1: zksyncTokens.usdc,
+    feeAmount: FeeAmount.LOWEST,
```
Comment (ChefMomota) [inline packages/farms/constants/324.ts:74]: pid 10 wrong token0, token1, feeAmount

File: packages/tokens/src/324.ts [hunk 6]
```diff
@@ -29,7 +29,15 @@ export const zksyncTokens = {
     '0x32Fd44bB869620C0EF993754c8a00Be67C464806',
     18,
     'rETH',
-    'RETH',
+    'Rocket Pool ETH',
     'https://www.paxos.com/busd/',
   ),
+  usdp: new ERC20Token(
```
Comment (ChefMomota) [inline packages/tokens/src/324.ts:35]: how about rename to `usdPlus` ?

**Pull request closed without merging**